README.md

```diff
@@ -74,6 +74,10 @@ If you know you will not be using the `$container` parameter or the `$getPreviou
     }
 ```
 
+A factory method returns the instance of the service it represents. A factory method does not need to ensure the unicity of the service returned. The container is in charge of calling the factory only once and then returning always the same service.
```

Comment:
I find it too complicated honestly (and too much about objects/services). Here is a try:

> Factory methods are only responsible for returning the value for the container entry. Nothing should be cached by service providers, this is the responsibility of the container.

Still not satisfied though…